cartography/intel/digitalocean/management.py

```diff
@@ -24,7 +24,7 @@ def sync(
 def sync_projects(
         neo4j_session: neo4j.Session,
         manager: Manager,
-        digitalocean_update_tag: str,
+        digitalocean_update_tag: int,
```

Comment:
The update tag really should be either string or int.
https://lyft.github.io/cartography/dev/writing-intel-modules.html#handling-cartography-s-update-tag